index.html

```diff
@@ -9,6 +9,7 @@
 		<meta name="description" content="En Estudio 11 pensamos que una comunidad de vecinos se tiene que gestionar con una visión global, facilitando la convivencia y comunicación fluida, optimizando los recursos y atendiendo el
 mantenimiento y calidad de los servicios, como si de una pequeña empresa se tratara.
 Sabemos que es imprescindible contar con una empresa profesional y con dedicación que se lleve a cabo la gestión económica, administrativa y de mantenimiento para conseguir una convivencia cómoda y de calidad." />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0">
```

Comment:
http://webdesign.tutsplus.com/es/articles/quick-tip-dont-forget-the-viewport-meta-tag--webdesign-5972